bin/score.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+
+
+def score_submission(predictions_path: str, status: str) -> dict:
+    """Determine the score of a submission. This is a placeholder function.
+
+    Args:
+        predictions_path (str): path to the predictions file
+        status (str): current submission status
+
+    Returns:
+        result (dict): dictionary containing score, status and errors
+    """
+    if status == "INVALID":
+        score_status = "INVALID"
+        score = None
+    else:
+        # placeholder file reading
+        with open(predictions_path, "r") as sub_file:
+            predictions_contents = sub_file.read()
+        try:
+            # placeholder scoring
+            score = 1 + 1
+            score_status = "SCORED"
+            message = ""
+        except Exception as e:
+            message = f"Error {e} occurred while scoring"
+            score = None
+            score_status = "INVALID"
+    result = {"score": {"auc": score, "status": score_status, "errors": message}}
+    return score_status, result
+
+
+def update_json(results_path: str, result: dict) -> None:
+    """Update the results.json file with the current score and status
+
+    Args:
+        results_path (str): path to the results.json file
+        result (dict): dictionary containing score, status and errors
+    """
+    with open(results_path, "r") as o:
+        data = json.load(o)
+    data.update(result)
```

Comment:
nit: on the off-chance that data is empty, will this fail?

Reply:
Yes, if the json file is empty it will fail. Whether I need to handle that scenario depends on what we decide to do below

Reply:
added in latest commit